submissions/kris-almort/html-css-popup/sass/partials/_header.scss

```diff
@@ -0,0 +1,102 @@
+.header {
+  display: flex;
+  justify-content: flex-end;
+  padding: 15px 0;
+  position: relative;
+  margin-bottom: 170px;
+}
+
+.top-menu {
+  display: flex;
+  padding: 0 15px;
+  align-items: center;
+}
+
+.top-menu__link-container {
+  line-height: 30px;
+  padding: 0 15px 0 0;
+  font-size: 13px;
+
+  &:last-child {
+    padding: 0;
+  }
+}
+
+.top-menu__link {
+  text-decoration: none;
+  transition: .1s;
+  color: rgba(0, 0, 0, 0.87);
+
+  &:hover {
+    text-decoration: underline;
+    opacity: .85;
+  }
+
+  &:focus {
+    outline-color: #e5e5e5;
+  }
+}
```

Comment:
Also, you should report to user his action is successful or not. For this use :active state to all interactive elements.

Reply:
Ok, thank 👍 Fixed in commit 377a172

Reply:
👍